drivers/media/mc/mc-request.c

```diff
@@ -504,3 +504,38 @@ void media_request_object_complete(struct media_request_object *obj)
 		media_request_put(req);
 }
 EXPORT_SYMBOL_GPL(media_request_object_complete);
+
```

Comment:
A link to the discussion on linux-media for this patch wouldn't go amiss.

Reply:
Discussion was on IRC